helm/hop-server/values.yaml

```diff
@@ -19,15 +19,30 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 
+## @param replicaCount number of replicas for the statefulset
 replicaCount: 1
 
+## @section Image
+## @param image.name Image name
+## @param image.pullPolicy Image pull policy
+## @param image.tag Visit: [Image tag](https://hub.docker.com/r/apache/hop/tags?page=1&ordering=last_updated)
+## @param image.env Environment variables
 image:
   name: docker.io/apache/hop
   pullPolicy: IfNotPresent
-  # Overrides the image tag whose default is the chart appVersion.
-  tag: "1.0-SNAPSHOT"
+  # Overrides the image tag
+  tag: "1.2.0"
```

Comment:
can we change this to "latest" so it will always be valid?

Reply:
It was an attempt to allow helm users to add the chart directly with `helm repo add [...]` without having to clone the whole repository, but it is finally not necessay, so you can delete it securely.

For the tag, we can indeed replace `1.2.0` by `latest`, as the helm user will still have the power to choose its release in its own values file

Reply:
Yes, that could be a pretty nice solution to publish the chart packaged to one of the helm repositories that exists. But I don't know central authority for Helm, only kind of private initiative (bitnami, oteemo...), or site-level repository (eg : [Apache Airflow](https://airflow.apache.org/docs/helm-chart/stable/index.html), [Gitea Chart](https://dl.gitea.io/charts/)).

Maybe the example of the ASF project Apache Airflow could be the simplest to implement ? Should I write this suggestion to a JIRA issue ?

Reply:
I'll take a look at how the Airflow team did it and contact them if needed, they are very helpful! 
Thanks for the great work and support!